theano/tensor/nnet/tests/test_nnet.py

```diff
@@ -1394,5 +1395,29 @@ def test_stabilize_log_softmax():
     rng = numpy.random.RandomState([2012, 8, 22])
     f(numpy.cast[config.floatX](rng.randn(2, 3)))
 
+
+def test_relu():
+    x = matrix('x')
+    rng = numpy.random.RandomState([2015, 6, 18])
```

Comment:
Use this seed instead (this allow to change it with a Theano flags)

theano.tests.unittest_tools.fetch_seed()